x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -31,6 +31,17 @@ import { hasDeleteAlertsCapability, hasSaveAlertsCapability } from '../../../lib
 
 const ENTER_KEY = 13;
 
+interface IAlertTypeState {
+  isLoading: boolean;
+  isInitialized: boolean;
+  data: AlertTypeIndex;
+}
+interface IAlertState {
+  isLoading: boolean;
+  data: Alert[];
+  totalItemCount: number;
+}
+
 export const AlertsList: React.FunctionComponent = () => {
```

Comment:
nit. but, we seem to be going down a road where some Interfaces have an *I* prefix and half don't.
Should we just decide if we want these prefixes or not? :)

Reply:
Yeah, I saw platform starting to do this (but not everywhere) so figured we may as well follow? :) After trying it out, I find it easier to figure out what is and isn't an interface

Reply:
Well, question is - does it matter what's an Interface as opposed to a type? If it does, can't we just use Intellisense to see that?

Reply:
Not really I guess, will revert

Reply:
haha cool :)